src/meltano/core/tracking/contexts/environment.py

```diff
@@ -27,7 +27,7 @@
 logger = get_logger(__name__)
 
 # This file is only ever created in CI when building a release
-release_marker_path = Path(__file__).parent / ".release_marker"
+release_marker_path = resources.files("meltano.core.tracking") / ".release_marker"
```

Comment:
I think this is fixing a bug in our tracking.

The old line seems to indicate that we were looking for `.release_marker` in `meltano/core/tracking/contexts`:

```python
>>> from pathlib import Path
>>> from meltano.core.tracking.contexts import __file__
>>> Path(__file__).parent
PosixPath('/Users/edgarramirez/Arch/meltano/src/meltano/core/tracking/contexts')
```

but at the same time, the build workflow puts it in `meltano/core/tracking`

https://github.com/meltano/meltano/blob/3db048d297c754ab24819b2bcc6e3286ffa44812/.github/workflows/build.yml#L52-L55